native/src/modules/common/renderHtml.js

```diff
@@ -137,8 +137,8 @@ const renderHtml = (html: string, files: PageResourceCacheStateType, theme: Them
         margin: 0;
         padding: 0;
 
+        font: -apple-system-body;
```

Comment:
Did you test whether this has any side effects on android?

Reply:
Yep, no side effects afaics. Feel free to test for yourself :)